src/w3c/templates/cgbg-headers.js

```diff
@@ -34,10 +34,10 @@ export default (conf, options) => {
               >
             </dd>`
         : ""}
-      ${"latestVersion" in conf // latestVersion can be falsy
+      ${conf.latestVersion !== null
         ? html`<dt>${l10n.latest_published_version}</dt>
             <dd>
-              ${conf.latestVersion
+              ${conf.latestVersion !== ""
```

Comment:
When is it `null`, and when is it `""` in particular?

Reply:
When it's null, it shouldn't be included... but when it's ""... I need to check :(